src/component/mxgraph/shape/task-shapes.ts

```diff
@@ -176,24 +176,84 @@ export class UserTaskShape extends BaseTaskShape {
   // use mxgraph svg2xml to generate the xml stencil and port it to code
```

Comment:
:warning: ref to the original icon must be updated here and in the documentation as well (bpm support page and perhaps in the README)